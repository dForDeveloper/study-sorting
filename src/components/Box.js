import React, { Component } from 'react';
import '../styles/main.scss';

function Box(props) {
  return (
    <div className={`${props.divClass}`} >
      <span className="Box--id">{props.id}</span>
    </div>
  );
}

export default Box;