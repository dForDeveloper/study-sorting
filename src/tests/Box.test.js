import React from 'react';
import Box from '../components/Box';
import { shallow } from 'enzyme';

describe('Box', () => {
  it('should match the snapshot', () => {
    let wrapper = shallow(
      <Box id="1" divClass="Box" key="0" />
    );
    expect(wrapper).toMatchSnapshot();
  });
});