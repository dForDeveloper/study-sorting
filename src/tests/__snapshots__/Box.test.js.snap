// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box should match the snapshot 1`] = `
<div
  className="Box"
>
  <span
    className="Box--id"
  >
    1
  </span>
</div>
`;
